### PR TITLE
Update Unity documentation

### PR DESCRIPTION
**Pull request by fuzzybinary**
### What does this PR do? What is the motivation?
Updates Unity setup documentation  with more information and corrects a few pieces.

Include a screenshot of the Unity setup UI.

### Merge instructions
- [x] Please merge after reviewing

### Additional notes

### CODE REVIEW

**Comment by drichards-87** [comment]
Created DOCS-8476 for Docs Team review.

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/unity.md [hunk 1]
```diff
@@ -73,11 +61,47 @@ You can control the data your application sends to Datadog RUM during instrument
...
+1. In Datadog, navigate to [**Digital Experience** > **Add an Application**][1].
+2. Choose `Unity` as the application type.
+3. Provide an application name to generate a unique Datadog application ID and client token.
+4. To disable automatic user data collection for either client IP or geolocation data, uncheck the boxes for those settings.
+
+To ensure the safety of your data, you must use a client token. For more information about setting up a client token, see the [Client Token documentation][2].
+
+### Specify Datadog settings in the Unity UI
+
+After installing the Datadog Unity SDK, you need to set Datadog's settings in the Unity UI. Navigate to your `Project Settings` and click on the `Datadog` section on the right hand side. You will see the following screen:
```
Comment (plousada) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/unity.md:75]: Shouldn't it be left hand side?

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/unity.md [hunk 2]
```diff
@@ -73,11 +61,47 @@ You can control the data your application sends to Datadog RUM during instrument
...
+
+{{<img src="real_user_monitoring/unity/datadog-setup-ui.png">}}
+
+The following parameters are available:
+
+| Parameter | Required? | Description |
+| --------- | --------- | ----------- |
+| Enable Datadog | No | Whether Datadog should be enabled. Disabling Datadog will cause any of the Datadog APIs to fail, throw exceptions, or return `null` from any calls, and will only stop the SDK from sending any information. |
```
Comment (plousada) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/unity.md:83]:
```suggestion
| Enable Datadog | No | Whether Datadog should be enabled. Disabling Datadog will not cause any of the Datadog APIs to fail, throw exceptions, or return `null` from any calls, and will only stop the SDK from sending any information. |
```

**Comment by plousada** [comment]
We should probably add a troubleshooting page with some common information regarding setting the tracking consent in order to see data, a statement saying we don't support the editor, consoles or desktop, and a snippet for setting SDK verbosity level.

**Review by plousada — CHANGES_REQUESTED** [review]

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/unity.md (hunk 1 above)
Comment (fuzzybinary) [reply content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/unity.md:75]: Yes.

**Comment by jhgilbert** [comment]
Hi @fuzzybinary, we don't describe future features or support in the docs, so I removed your note that said "Datadog plans on adding support for Open UPM after Beta." But I want to make sure to address the underlying problem you were trying to solve. Is there something we can say about the present situation (xyz is not supported) to make the situation clear to the user?

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/unity.md
Comment (jhgilbert) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/unity.md:1]: @fuzzybinary, did you mean to link to this page from the other page?

Comment (fuzzybinary) [reply content/en/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/unity.md:1]: Ah, it should be included on the overall "Troubleshooting" if that's what you mean, adding "Unity" as one of the tabs. I'm actually not 100% sure where that lives.

**Comment by fuzzybinary** [comment]
> But I want to make sure to address the underlying problem you were trying to solve. Is there something we can say about the present situation (xyz is not supported) to make the situation clear to the user?

There's nothing we can add like that no. Open UPM is just a package repository, we just haven't yet.

File: content/en/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/unity.md [hunk 3]
```diff
@@ -23,28 +23,25 @@ further_reading:
...
 
-If you're not seeing any data in Datadog, first make sure you are running your app on an iOS or Android simulator, emulator, or device, and not from the editor.
+If you're not seeing any data in Datadog:
 
-<div class="alert alert-info">
-Datadog does not currently support sending data fro the Unity Editor, only from iOS and Android simulators, emulators, and devices.
-</div>
-
-Next, check that you have set the `TrackingConsent` as part of your initialization. Tracking consent is set to `TrackingConsent.Pending` during initialization,
+1. Make sure you are running your app on an iOS or Android simulator, emulator, or device, and not from the editor. Datadog does not support sending data from the Unity Editor, only from iOS and Android simulators, emulators, and devices.
```
Comment (fuzzybinary) [inline content/en/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/unity.md:40]: Is there any way we can call more attention to the lack of Editor support. This is a common issue we've had and I don't want it missed.

Comment (jhgilbert) [reply content/en/real_user_monitoring/mobile_and_tv_monitoring/troubleshooting/unity.md:40]: Sure, I'll put it in an alert

**Review by jhgilbert — APPROVED** [review]

**Comment by jhgilbert** [comment]
@plousada, did you want to take another look?

**Review by plousada — APPROVED** [review]

**Pull request merged**